core/src/main/java/com/novoda/noplayer/internal/exoplayer/AvailableAdverts.java

```diff
@@ -35,4 +35,16 @@ static void markAllFutureAdvertsAsAvailable(long currentPositionInMillis, List<A
             }
         }
     }
+
+    static void markAllFutureAdvertsAsPlayed(long currentPositionInMillis, List<AdvertBreak> advertBreaks, AdPlaybackState adPlaybackState) {
```

Comment:
This is in the wrong class. Or at least, the name of this class now needs to change. The adverts wouldn't be available if they are marked as `Played`.

Reply:
Ah wait. This isn't used...

Reply:
4fd5fddb

Reply:
I started doing it, but never used :man_facepalming: